index.html

```diff
@@ -38,150 +38,36 @@ <h1>
-                <div class="grid-item">
-                    <h2>Multi-Post Stories Gain+Glory</h2>
-                </div>
-                <div class="grid-item">
-                    <ul>
-                        <li> Ruby on Rails</li>
-                        <li> CSS </li>
-                        <li> JavaScript </li>
-                        <li> HTML </li>
-                    </ul>
-                </div>
-                <div class="grid-item">
-                    <button class="normal-btn">
-                        See Project
-                    </button>
-                </div>
-            </div>
-        </div>
-        <div class="background">
-            <div class="grid">
-                <div class="grid-item">
-                    <h2>Multi-Post Stories Gain+Glory</h2>
-                </div>
-                <div class="grid-item">
-                    <ul>
-                        <li> Ruby on Rails</li>
-                        <li> CSS </li>
-                        <li> JavaScript </li>
-                        <li> HTML </li>
-                    </ul>
-                </div>
-                <div class="grid-item">
-                    <button class="normal-btn">
-                        See Project
-                    </button>
-                </div>
-            </div>
-        </div>
-        <div class="background">
-            <div class="grid">
-                <div class="grid-item">
-                    <h2>Multi-Post Stories Gain+Glory</h2>
-                </div>
-                <div class="grid-item">
-                    <ul>
-                        <li> Ruby on Rails</li>
-                        <li> CSS </li>
-                        <li> JavaScript </li>
-                        <li> HTML </li>
-                    </ul>
-                </div>
-                <div class="grid-item">
-                    <button class="normal-btn">
-                        See Project
-                    </button>
-                </div>
-            </div>
-        </div>
-        <div class="background">
-            <div class="grid">
-                <div class="grid-item">
-                    <h2>Multi-Post Stories Gain+Glory</h2>
-                </div>
-                <div class="grid-item">
-                    <ul>
-                        <li> Ruby on Rails</li>
-                        <li> CSS </li>
-                        <li> JavaScript </li>
-                        <li> HTML </li>
-                    </ul>
-                </div>
-                <div class="grid-item">
-                    <button class="normal-btn">
-                        See Project
-                    </button>
-                </div>
-            </div>
-        </div>
-    </div>
+    
+    <section class="portfolio" id="portfolio">
+        <h2>My Recent Works</h2> 
     </section>
+
+    <div class="works">
+    
+    </div>
+
+    <div class="project-pop">
+            
+    </div>
```

Comment:
- [ ] You have done a very great job in implementing the changes so far, :clap: However, Your Cards are not loaded  in your browser, Please resolve this :slightly_smiling_face: 

Below is an illustration of the difference between yours and the design: :arrow_down:

<table>
    <tr>
      <td>Your Design</td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

![Student Works](https://github.com/Arch-Noize/portfolio-microverse/assets/43528594/b276c3d6-0c3b-4fa5-9f02-7d11903d8d65)

</td>
      <td>

![Figma Works](https://github.com/Arch-Noize/portfolio-microverse/assets/43528594/3e8fd8e4-4e4d-444c-97f3-d1044cca7e95)

</td>
    </tr>
  </table>

Reply:
Hello, thank you! I'm confused though. The cards are loading on my end...
![image](https://github.com/Arch-Noize/portfolio-microverse/assets/91759364/8e86eb70-2db6-47cb-bf30-db33ac09240b)
![image](https://github.com/Arch-Noize/portfolio-microverse/assets/91759364/64ff6fd9-3c5b-41dd-922c-93fc6f2a883d)

Could you send me a screenshot of possible errors please? Those two div tags are what the .js file is targeting to print the cards

Reply:
Hello @Arch-Noize, If it works on your end, It must be that you forgot to push your changes from the local repository. Please check that out and let me know if that is the case. :slightly_smiling_face:

Reply:
I just double-checked. The version I have locally is the same as the one I pushed. I am so confused hahaha. Sorry for the inconvenience.